dbPedia_18thCentComposers.js

```diff
@@ -0,0 +1,146 @@
+const http = require('http');
+const express = require('express');
+const fs = require('fs');
+const request = require('request');
+const cheerio = require('cheerio');
+const app = express();
+const sleep = require('system-sleep');
+const async = require('async');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer(function(request, response) {
+    response.statusCode = 200;
+    response.setHeader('Content-Type', 'text/plain');
+    response.end('Hello World\n');
+});
+
+server.listen(port, hostname, function() {
+    console.log(`Server running at http://${hostname}:${port}/`);
+
+    var elemNum = 0;
+
+    var newObj = {
+        table: []
+    };
+
+    fs.readFile('dbPedia_18thCentComposers.json', 'utf-8', function(err, array) {
+        if (err) throw err
+        newObj.table.push({
+            name: "dummyname",
+            birthDate: "13-1-1689",
+            birthPlace: "dummyPlace",
+            deathDate: "dummyDeathDate",
+            deathPlace: "dummyDeathPlace ",
+            Pseudonym: "dummyPseudonym ",
+            works: "dummyWorks"
+        });
+        var json = JSON.stringify(newObj);
+        fs.writeFile('myjsonfile2.json', json, 'utf8');
+    });
+
+    function getResults(queryURL, callback) {
+        var name = 'done'
+        request(queryURL, function(error, response, html) {
+            if (html) {
+                var $ = cheerio.load(html);
+                if (html != null) {
+                    //name
+                    name = $('span[property="dbp:name"]').text();
+                    if (!name) {
+                        var name = ($('h1 a').text()).split("(");
+                    }
+                    newObj.table.push({
+                        name: name
+                    });
+                    //date of birth
+                    dateOfBirth = $('span[property="dbp:birthDate"]').text();
+                    if (!dateOfBirth) {
+                        dateOfBirth = "xxxx-xx-xx";
```

Comment:
This is the same code as above in the other file. Please consider merging the two into a dedicated class that provides that functionality.